rslib/src/backend/mod.rs

```diff
@@ -426,6 +432,18 @@ impl BackendService for Backend {
         })
     }
 
+    fn negate_search(&self, input: pb::String) -> Result<pb::String> {
```

Comment:
Since these don't require the collection and should complete quickly, we can add them to want_release_gil() in rsbridge